### ISSUE TITLE
Unit 'DECCipherBase' not found

### ISSUE THREAD

**Issue by djdance** [MHumm/DelphiEncryptionCompendium#29]
I have just installed the library via Getit Package Manager on Delphi 10.3 Version 26.0.36039.7899 
Restarted IDE
But able compile nothing: 

**[dcc32 Fatal Error] test.pas(8): F2613 Unit 'DECCipherBase' not found.**

Could you help, what should I do to next?  Thanks

**Comment by MHumm** [comment]
Oh dear, looks like the GetIt provided "Lite" version has a reference to a file included which shall not be part of the Lite version.
Is test.pas from DEC? I doubt that. Can you tell me which unit refers to DECCipherBase?

Ok, I'll have to update DEC Lite in GetIt, but to be honest: if I'd be you I'd use the full version. The Lite version does not contain any encryption code because Embarcadero doesn't want to have any trouble regarding US export regulations for cryptography code. I'd do these steps if I were you:

1.  Remove that Lite version.
2. Download the last release 6.3 from here, it's newer than the 6.2.x provided by GetIt:
    [https://github.com/MHumm/DelphiEncryptionCompendium/releases](https://github.com/MHumm/DelphiEncryptionCompendium/releases)
3. Unzip it into an empty folder
4. Open the SetIDEPaths project from the Install subdirectory and run it
5. Restart the IDE
6. Open the DEC60 project group from the source folder and compile the DEC60 project.
    DEC should be ready to use now. Then look at the demos and read the documentation
    to get a start. If you have any further questions after that ask.

**Comment by MHumm** [comment]
I submitted a 6.3 Lite for GetIt now, which should fix this. It may take a few days though until it appears.

**Comment by MHumm** [comment]
Did the provided steps solve your issue?

**Comment by djdance** [comment]
a) I can set up fill version, but my goal is to use GetIt as far as possible (as a part of modern world: manual installing is so 2000-th). But I stored your advice in bookmarks, thanks!

b)

There are still no Lite 6.3 in GetIt, so I am continue to waiting, and thank you for you care!
(I tried to uninstall 6.2.1 before. BTW how does Embarcadero work with updates, do they approve them or review somekind, like Apple?)

![image](https://user-images.githubusercontent.com/2269720/134038116-6a1ff3b5-90aa-4502-88a5-d2ee07bc339c.png)

**Comment by MHumm** [comment]
I can fully understand your frustration. What I've heared is, that there are some organizational issues. As it is currently implemented submitting an update requires me to fill in the same form as a first entry, so I have to fill in everything again.
But I also heared, that they're working on some server side improvements which will allow people to update their already submitted and approved libraries easily themselves. 
About approval: they most likely do check the contents of the form and they won't accept cipher algorithms in order to not run into any problems with US export regulations. This is the only reason for DEC Lite's existence. Hash algorithms are ok, as they don't encrypt any message...
